src/index.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const state = {
+  temperature: 79,
+  city: 'London',
+  isF: true,
+};
+
+const convertKtoC = (temp) => temp - 273.15;
+const convertKtoF = (temp) => (temp - 273.15) * (9 / 5) + 32;
+const convertCtoF = (temp) => temp * (9 / 5) + 32;
+const convertFtoC = (temp) => (temp - 32) * (5 / 9);
+
+const getLatAndLon = (city) => {
+  return axios
+    .get('http://127.0.0.1:5000/location', {
```

Comment:
Consider using a constant variable like `LOCATION_IQ_URL` here instead of a string literal for the URL.